### PR TITLE
[zh] remove redundant content

### PR DESCRIPTION
**Pull request by DangHT**
[Kubernetes 组件](https://kubernetes.io/zh/docs/concepts/overview/components/)页上有重复的内容

### CODE REVIEW

**Comment by DangHT** [comment]
#25065

File: content/zh/docs/concepts/overview/components.md [hunk 1]
```diff
@@ -22,17 +22,6 @@ card:
   weight: 20
 -->
 
-<!--
-When you deploy Kubernetes, you get a cluster.
-{{< glossary_definition term_id="cluster" length="all" prepend="A Kubernetes cluster consists of">}}
```
Comment (tengqm) [inline content/zh/docs/concepts/overview/components.md:27]: Do not remove this paragraph, instead, change line 27 to :

```
{{</* glossary_definition term_id="cluster" length="all" prepend="A Kubernetes cluster consists of" */>}}
```

Comment (tengqm) [reply content/zh/docs/concepts/overview/components.md:27]: Please also help fix line 195-196 where the word "Kubernetes" is broken

Comment (DangHT) [reply content/zh/docs/concepts/overview/components.md:27]: done

**Comment by tengqm** [comment]
preview: https://deploy-preview-25093--kubernetes-io-master-staging.netlify.app/zh/docs/concepts/overview/components/
/lgtm
/approve

**Pull request merged**